<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blackjack</class>
 <widget class="QDialog" name="blackjack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>1081</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="dealerCardsLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>1081</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="playerCardsLayout"/>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>280</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayerScore">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Twój wynik</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDealerScore">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wynik dealera</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResult">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rezultat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>390</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>500</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyjście</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
